src/main/java/tools/jackson/databind/ext/javatime/deser/JavaTimeDeserializerModifier.java

```diff
@@ -17,9 +12,7 @@ public JavaTimeDeserializerModifier() { }
     @Override
     public ValueDeserializer<?> modifyEnumDeserializer(DeserializationConfig config, JavaType type,
             BeanDescription.Supplier beanDescRef, ValueDeserializer<?> defaultDeserializer) {
-        if (type.hasRawClass(Month.class)) {
```

Comment:
We can remove this class altogether (I'll do that)

Reply:
Thanksss I was actually 50/50 on this 😅 forgot to mention.